api/src/Controller/BaseController.php

```diff
@@ -454,53 +455,19 @@ public function arrayResponse(Request $request, Response $response, $data, $code
 
     public function getDepartment($id)
     {
-        $sql = <<<SQL
-            SELECT
-                *,
-                (
-                SELECT
-                    COUNT(DepartmentID)
-                FROM
-                    `Departments` d2
-                WHERE
-                    d2.ParentDepartmentID = d1.DepartmentID AND NOT NAME = 'Historical Placeholder'
-            ) AS childCount,
-            (
-                SELECT
-                    GROUP_CONCAT(Email)
-                FROM
-                    EMails
-                WHERE
-                    DepartmentID = d1.DepartmentID
-            ) AS email
-            FROM
-                Departments d1
-            WHERE
-SQL;
+        $select = Select::new($this->container->db);
+        $select->columns('*');
+        $select->columns($select->subselect()->columns('COUNT(DepartmentID)')->from('`Departments` d2')->whereEquals(['d2.ParentDepartmentID' => 'd1.DepartmentID'])->andWhere("NOT NAME = 'Historical Placeholder'")->as('childCount')->getStatement());
```

Comment:
Including the alias in the from arguments is fine IMO, but you could also say, I believe, `->from('Departments')->as('d2')`. This is not a change you need to make here. This is just informational.